app/models/messaging_preference.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+require 'common/models/base'
+# Secure Messaging Notification Preference Model
+class MessagingPreference < Common::Base
+  FREQUENCY_UPDATE_MAP = {
+    'none' => 0,
+    'each_message' => 1,
+    'daily' => 2
+  }.freeze
+
+  FREQUENCY_GET_MAP = {
```

Comment:
I thought about that...but yet another vets-api request round trip for the frontend to fetch the values - unless we return it as metadata in every "get preferences" request. 

I also thought about putting constants in vets-json-schema to facilitate frontend-backend consistency, but with only three values it did not seem worth the effort. Open to reconsidering though.